.codebuild/buildspec.mftf.tests.prs.yml

```diff
@@ -0,0 +1,36 @@
+version: 0.2
+env:
+  parameter-store:
+    MAGENTO_BACKEND_URL: "/pwa/MAGENTO_BACKEND_URL"
+    MAGENTO_ADMIN_USERNAME: "/pwa/MAGENTO_ADMIN_USERNAME"
+    MAGENTO_ADMIN_PASSWORD: "/pwa/MAGENTO_ADMIN_PASSWORD"
+    STOREFRONT_URL: "/pwa/STOREFRONT_URL_STAGING"
+
+phases:
+  pre_build:
+    commands:
+      - echo copying env vars to env file
+      - touch .env
+      - echo MAGENTO_BASE_URL=$MAGENTO_BACKEND_URL >> .env
+      - echo MAGENTO_ADMIN_USERNAME=$MAGENTO_ADMIN_USERNAME >> .env
+      - echo MAGENTO_ADMIN_PASSWORD=$MAGENTO_ADMIN_PASSWORD >> .env
+      - echo MODULE_WHITELIST=Pwa >> .env
+      - echo STOREFRONT_URL=$STOREFRONT_URL >> .env
+      - echo SELENIUM_HOST=selenium >> .env
+      - mv .env dev/tests/acceptance
+
+  build:
+    commands:
+      - echo build Docker image on `date`
+      - docker-compose build
+      - nohup docker-compose up & ./.bin/kill-selenium
+
+  post_build:
+    commands:
+      - echo build Docker image complete `date`
+      - mv consolidated_test_report ./dev/tests/acceptance/tests/_output
```

Comment:
curious, do you need the `mv` here? from what I am understanding from your script, you want to navigate into the `_output` folder and then call `allure serve allure-results/`. Am I missing what `mv` is doing here?

Reply:
M moving to it output directory and next steps are as mentioned in first comment.